src/app/core/common-components/entity-select/entity-select.component.html

```diff
@@ -16,11 +16,13 @@
       [createOption]="disableCreateNew ? null : createNewEntity"
     >
       <ng-template let-item>
-        <app-entity-block
-          style="margin: auto"
-          [entity]="item"
-          [linkDisabled]="form.enabled"
-        ></app-entity-block>
+        @if (!item.isHidden) {
```

Comment:
couldn't / shouldn't we do this condition within the auto-complete component instead of just here?